backend/package.json

```diff
@@ -19,7 +19,7 @@
   ],
   "engineStrict": true,
   "engines": {
-    "node": "^16.16"
+    "node": "=18.14.1"
```

Comment:
Do we need to update the frontend/package.json too? It's still on 16.16.

Reply:
yeah probably should. It wasn't failing checks on the frontend, oddly